proofOC/bookRoom.py

```diff
@@ -1,6 +1,5 @@
 # https://kivy.org/doc/stable/guide/packaging-osx.html
 
-import logging
```

Comment:
This needs to be kept in for the logger to be created. The program does not run otherwise:

```
 Traceback (most recent call last):
   File "/Users/rpalko/Documents/temple/libraryroomscheduler/proofOC/bookRoom.py", line 201, in <module>
     logger = logging.getLogger("appLog")
 NameError: name 'logging' is not defined
```